### PR TITLE
Fix validate unicode character in colname

### PR DESCRIPTION
**Pull request by jinchengchenghh**

### CODE REVIEW

**Review by rui-mo — APPROVED** [review]

File: velox/substrait/SubstraitToVeloxPlanValidator.cpp [hunk 1]
```diff
@@ -806,6 +822,15 @@ bool SubstraitToVeloxPlanValidator::validate(
...
     }
   }
+  if (sRead.has_base_schema()) {
+    const auto& baseSchema = sRead.base_schema();
+    if (!validateColNames(baseSchema)) {
+      std::cout
+          << "Validation failed for column name contains illegal charactor."
+          << std::endl;
+      return false;
+    };
```
Comment (rui-mo) [inline velox/substrait/SubstraitToVeloxPlanValidator.cpp:832]: Looks `;` could be removed?

Comment (jinchengchenghh) [reply velox/substrait/SubstraitToVeloxPlanValidator.cpp]: Fixed

**Comment by jinchengchenghh** [comment]
Can you help start the CI again? @rui-mo

**Comment by rui-mo** [comment]
> Can you help start the CI again? @rui-mo

Restarted.

**Pull request merged**